### PR TITLE
Add module augmentation

### PR DESCRIPTION
**Pull request by fgiova**
Simply add typescript module augmentation

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -5,6 +5,12 @@ import {
   LightMyRequestResponse,
 } from 'fastify'
 
+declare module 'fastify' {
+	interface FastifyInstance {
+		restart: () => Promise<void>
```
Comment (Uzlopak) [inline index.d.ts:10]: Nitpick: we use 2 spaces as spacing.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/restartable",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (Uzlopak) [inline package.json:3]: Please revert that.

File: index.d.ts (hunk 1 above)
Comment (fgiova) [reply index.d.ts]: Sorry bad setting on IDE... now is set to 2 spaces

**Comment by fgiova** [comment]
@Uzlopak the only way for pass test is add "@ts-ignore" on augmentation. The switch skipLibCheck on tsd seem not work. Is acceptable as workaround?

**Comment by Uzlopak** [comment]
install @types/node as dev dependency.

**Comment by Uzlopak** [comment]
Yes, tested it locally. Installing the types will fix the issue. Had the same problem few days ago.

What is missing is still a test for this change.

I would suggest to add in line 7 of  index.test-d.ts the following line:


`expectType<() => Promise<void>>(app.restart)`

image

And then it should be mergeable.

**Comment by fgiova** [comment]
Tnx so much... I' m totally noob with tsd.
> Yes, tested it locally. Installing the node-types will fix the issue. Had the same problem few days ago.
> 
> What is missing is still a test for this change.
> 
> I would suggest to add in line 7 of index.test-d.ts the following line:
> 
> `expectType<() => Promise<void>>(app.restart)`
> 
> image
> 
> And then it should be mergeable.

**Comment by Uzlopak** [comment]
@mcollina 
PTAL

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**